### ISSUE TITLE
Can't disable coloured output

### ISSUE THREAD

**Issue by alexjurkiewicz** [Pryz/yaml-lint#12]
This is a problem in eg build script logging because you get ugly output like this in plain text:

```
yaml-lint ./global.yaml 
Checking the content of ["./global.yaml"]
[32mFile : ./global.yaml, Syntax OK[0m
Done.
```

**Comment by aerostitch** [comment]
#15 adds that ability.

**Comment by aerostitch** [comment]
Released as part of 0.0.10 on rubygems.org

**Comment by alexjurkiewicz** [comment]
thanks!!
